.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,18 @@
+name: Pull Request Labeler
+
+on:
+  pull_request:
+    types: opened
+
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
```

Comment:
is there a reason you are checking out the code?  I would remove this step unless you plan to make use of a separate small repo that holds all your automation scripts, etc.